axelor-account/src/main/java/com/axelor/apps/account/web/FixedAssetController.java

```diff
@@ -618,4 +620,32 @@ protected Map<String, Object> buildWizardValues(Context context) {
     fieldMap.put("saleTaxLineSet", saleTaxLineSet);
     return fieldMap;
   }
+
+  @ErrorException
+  public void setDomainAnalyticDistributionTemplate(ActionRequest request, ActionResponse response)
+      throws AxelorException {
+    Context context = request.getContext();
+    FixedAsset fixedAsset = context.asType(FixedAsset.class);
+
+    String domain =
```

Comment:
Can you put all the domain computation in the attrsService please, so in the controller we just got : 
response.setAttr("analyticDistributionTemplate", "domain", Beans.get(FixedAssetAttrsService.class)
              .addCurrentAnalyticDistributionTemplateInDomain(fixedAsset))